README.md

```diff
@@ -29,22 +31,20 @@ The rendered final report is located [here](https://github.com/UBC-MDS/olympic_m
 
 The analysis can be reproduced by following these steps:
 
-1. Clone the GitHub repository and install the required [dependencies](#dependencies)
-2. Navigate into the root directory of the cloned repository on your local machine
-3. Run the `Makefile` file with the following command:
+1.  Clone the GitHub repository and install the required [dependencies](#dependencies)
+2.  Navigate into the root directory of the cloned repository on your local machine
+3.  Run the `Makefile` file with the following command:
 
+```{=html}
+<!-- -->
 ```
-make all
-```
+    make all
 
 The local repository can be reset to the initial cloned state by running:
 
-```
-make clean
-```
-
+    make clean
```

Comment:
This one might also need to be surrounded by triple ticks, I can't preview the rendered version so these comments may or may not be relevant